mesonbuild/interpreter.py

```diff
@@ -3226,6 +3229,45 @@ def func_configure_file(self, node, args, kwargs):
             self.build.data.append(build.Data([cfile], idir))
         return mesonlib.File.from_built_file(self.subdir, output)
 
+    @permittedKwargs(permitted_kwargs['include'])
+    def func_include(self, node, args, kwargs):
+        self.validate_arguments(args, 1, [str])
+        mesonlib.check_direntry_issues(args)
+        for i in mesonlib.extract_as_list(kwargs, 'if_found'):
+            if not hasattr(i, 'found_method'):
+                raise InterpreterException('Object used in if_found does not have a found method.')
+            if not i.found_method([], {}):
+                return
+        if os.path.isabs(args[0]):
+            absname = os.path.abspath(args[0])
+        else:
+            absname = os.path.abspath(os.path.join(self.environment.get_source_dir(), args[0]))
+        prev_current_file = self.current_file
+        if absname in self.included_files:
+            raise InvalidArguments('Tried to include file "%s", which has already been included.'
+                                   % absname)
+        self.included_files.add(absname)
+        if not absname in self.build_def_files:
```

Comment:
__[Flake8]__

[E713] test for membership should be 'not in'


<!-- Comment by [SideCI](https://sideci.com) -->

Reply:
Sorry forgot to run flake8